hip-service/Discovery/Patient/Database/Migrations/20200116095059_AddDiscoveryRequestTable.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace hip_service.Discovery.Patient.Database.Migrations
+{
+    public partial class AddDiscoveryRequestTable : Migration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4833676)